Packs/AzureDevOps/Integrations/AzureDevOps/AzureDevOps_test.py

```diff
@@ -822,3 +825,1006 @@ def test_generate_login_url(mocker):
+    from AzureDevOps import Client, pull_request_reviewer_list_command
+
+    authorization_url = 'https://login.microsoftonline.com/organizations/oauth2/v2.0/token'
+    requests_mock.post(authorization_url, json=get_azure_access_token_mock())
+
+    project = 'test'
+    repository = 'xsoar'
+    pull_request_id = pull_request_id
+
+    url = f'https://dev.azure.com/{ORGANIZATION}/{project}/_apis/git/repositories/{repository}/' \
+          f'pullRequests/{pull_request_id}/reviewers'
+
+    mock_response = json.loads(load_mock_response(mock_response_path))
+    requests_mock.get(url, json=mock_response)
+
+    client = Client(
+        client_id=CLIENT_ID,
+        organization=ORGANIZATION,
+        verify=False,
+        proxy=False,
+        auth_type='Device Code')
+
+    result = pull_request_reviewer_list_command(client, {'pull_request_id': pull_request_id}, ORGANIZATION, repository, project)
+
+    assert result.outputs_prefix == 'AzureDevOps.PullRequestReviewer'
+    assert result.readable_output.startswith("### Reviewers List")
+
+
+@pytest.mark.parametrize('pull_request_id, mock_response_path', [('42', 'pull_request_not_found.json')])
+def test_2_azure_devops_pull_request_reviewer_list_command(requests_mock, pull_request_id: str, mock_response_path: str):
+    """
+    Given:
+     - pull_request_id (pr does not exist)
+    When:
+     - executing azure-devops-pull-request-reviewer-list command
+    Then:
+     - Ensure outputs_prefix and readable_output are set up right
+     - Ensure informative message when pull_request_id does not exist
+    """
+    from AzureDevOps import Client, pull_request_reviewer_list_command
+
+    authorization_url = 'https://login.microsoftonline.com/organizations/oauth2/v2.0/token'
+    requests_mock.post(authorization_url, json=get_azure_access_token_mock())
+
+    project = 'test'
+    repository = 'xsoar'
+    pull_request_id = pull_request_id
+
+    url = f'https://dev.azure.com/{ORGANIZATION}/{project}/_apis/git/repositories/{repository}/' \
+          f'pullRequests/{pull_request_id}/reviewers'
+
+    mock_response = json.loads(load_mock_response(mock_response_path))
+    requests_mock.get(url, json=mock_response)
+
+    client = Client(
+        client_id=CLIENT_ID,
+        organization=ORGANIZATION,
+        verify=False,
+        proxy=False,
+        auth_type='Device Code')
+
+    result = pull_request_reviewer_list_command(client, {'pull_request_id': pull_request_id}, ORGANIZATION, repository, project)
+
+    assert 'The requested pull request was not found.' in result.raw_response.get("message")
+
+
+@pytest.mark.parametrize('args, organization, expected_result',
+                         [({'organization_name': 'OVERRIDE'}, 'TEST', 'OVERRIDE'),
+                          ({}, 'TEST', 'TEST'),
+                          ({'organization_name': 'OVERRIDE'}, '', 'OVERRIDE')])
+def test_1_organization_repository_project_preprocess_function(args: dict, organization: str, expected_result: str):
```

Comment:
please rename all tests according to what they test. 
for a shorter name, the meaning of the test is more important than including the complete function name